docs/install/install.rst

```diff
@@ -1,51 +1,71 @@
 .. meta::
-  :description: TransferBench documentation 
+  :description: TransferBench documentation
   :keywords: TransferBench, API, ROCm, HIP
 
+.. _install-transferbench:
+
 ---------------------------
-TransferBench installation
+Installation
 ---------------------------
 
-The following software is required to install TransferBench:
+This document provides information required to install and build TransferBench.
 
-* ROCm stack installed on the system (HIP runtime)
-* `libnuma` installed on the system
+Prerequisite
+---------------
 
---------------------------
-Building TransferBench
---------------------------
+* Install ROCm stack on the system (HIP runtime)
+* Install ``libnuma`` on the system
+* Enable AMD IOMMU and set to passthrough for AMD Instinct cards
 
-To build TransferBench using Makefile, use the following instruction:
+Build TransferBench
+---------------------
+
+To build TransferBench using Makefile, use:
 
 .. code-block:: bash
 
             $ make
 
-To build TransferBench using CMake, use the following commands:
+To build TransferBench using CMake, use:
 
 .. code-block:: bash
 
                 $ mkdir build
-    
+
                 $ cd build
-    
+
                 $ CXX=/opt/rocm/bin/hipcc cmake ..
-    
+
                 $ make
 
-.. Note:: 
+.. note::
 
-If ROCm is installed in a folder other than `/opt/rocm/`, set `ROCM_PATH` appropriately.
+  If ROCm is installed in a folder other than ``/opt/rocm/``, set ``ROCM_PATH`` appropriately.
+
+Build documentation
+-----------------------
+
+To build documentation locally, use:
+
+.. code-block:: bash
+
+  cd docs
+  pip3 install -r .sphinx/requirements.txt
+  python3 -m sphinx -T -E -b html -d _build/doctrees -D language=en . _build/html
 
---------------------------
 NVIDIA platform support
 --------------------------
 
-TransferBench may also be built to run on NVIDIA platforms via HIP, but requires a HIP-compatible CUDA version installed. For example, CUDA 11.5.
+You can build TransferBench to run on NVIDIA platforms via HIP or native NVCC.
```

Comment:
```suggestion
You can build TransferBench to run on NVIDIA CUDA platforms via HIP or native NVCC.
```

Reply:
The HIP should be hyperlinked. 
You should also describe what is.

Reply:
Hyperlinked HIP

Reply:
This has been updated as per latest review from dev.